core/js/typespecs/app_types.ts

```diff
@@ -328,9 +347,16 @@ export const roleSchema = z
     description: roleBaseSchema.shape.description,
     deleted_at: roleBaseSchema.shape.deleted_at,
     member_permissions: z
-      .array(permissionEnum)
+      .array(
```

Comment:
This is the only change. The rest is just re-ordering.

Reply:
is there some back compat issues we need to be careful about?

Reply:
Nothing in the SDK or backend directly uses the roles API (just forwarding things along to the webapp).

It is true that if someone was using the roles REST API on an old backend it will break once it gets to the webapp, but I think that's okay since roles are still an in-development feature.